release_instructions/ReleaseInstructions.md

```diff
@@ -86,7 +86,7 @@
 ## Conda-Forge Release
 
 After GitHub & PyPi release, conduct release on Conda-Forge
-* [TODO] Add Conda-Forge release steps.
+* Please refer to [conda release instructions](https://github.com/autogluon/autogluon/blob/master/release_instructions/update-conda-recipes.md) for details.
```

Comment:
There is a section below titled "Post Release Conda-Forge Patching", could you add some notes there as well, or a link if we have a relevant doc? Thanks!

Reply:
Also, you can do a relative link, this will be better.

For example, in README.md in root folder, I use this to link to AWESOME.md:

```
A comprehensive list is available [here](AWESOME.md#videos--tutorials)
```

I believe you can do this:

```
[conda release instructions](update-conda-recipes.md)
```

Reply:
This is better! Just updated.